pkg/run/git_exec.go

```diff
@@ -0,0 +1,44 @@
+package run
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func GetBranchName() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD")
```

Comment:
@ozamosi does it do the same thing as getting the branch name with go-git for the bootstrap wizard? What is the preferred way to get the branch name: this one or what we used in the wizard?

Reply:
Let's discuss!